packages/gestalt-codemods/generic-codemods/entry.sh

```diff
@@ -9,7 +9,7 @@ if [[ $# -eq 0 ]]; then
   exit 0
 fi
 
-yarn jscodeshift --ignore-pattern=**/node_modules/** --ignore-pattern=build/** --parser=tsx -t="${DIR}"/"${1}".tsx "${@:2}"
+yarn jscodeshift --ignore-pattern=**/node_modules/** --ignore-pattern=build/** --parser=tsx -t="${DIR}"/"${1}".ts "${@:2}"
```

Comment:
The `yarn codemod detectManualReplacement` command was failing since previously it was appending `.tsx` instead of `.ts` to the transform file.

Reply:
OMG thank you fixing this